features/FxA-89-devices-view/README.md

```diff
@@ -0,0 +1,410 @@
+
+# Devices View
+
+Feature card: https://github.com/mozilla/fxa-features/issues/8
+
+Original draft: https://docs.google.com/document/d/1VrysJ4LFPZ20IeMMmRts5l_2GUHhm890rlQhD012U_g
+
+Engineering bugs: https://waffle.io/mozilla/fxa?milestone=FxA-89:%20devices%20view
+
+## Stories
+
+* So that I can best ensure the integrity of my Sync setup,
+  as a Firefox Sync user,
+  I want to see information about the devices
+  connected to my Firefox Account.
+* So that I can obtain a device overview more quickly,
+  I want the devices connected to my account
+  to have recognizable names by default.
+* I want to easily see the type
+  and browser details of the each device, 
+* So that I can better tell what device I am currently using,
+  I want it to be easily distinguishable from the other items.
+* So that I can easily detect when a device is no longer active,
+  I want to be able to see when my devices last authenticated.
+
+
+## Hypothesis
+
+1. If we add a device view in settings
+   for Firefox Account users with multiple devices,
+   then **we will decrease the number of malicious connections**
```

Comment:
It seems like `malicious connections` should be something along the lines of `active malicious connections`. The feature won't block new malicious connections from being made, but it will offer a remedy for connections already made.